src/ib_module.F

```diff
@@ -1713,7 +1713,7 @@ subroutine ib_lwr_flx(stag,ix,jy,kz,c1,c2,rrw,dumz,a,kbdy,vadvorder)
 
   ELSEIF( stag.eq.4 )THEN
 
-    !$acc parallel loop private(i,j,k,kval,wbar)
+    !$acc parallel loop default(present) private(i,j,k,kval,wbar)
```

Comment:
maybe add `gang vector` clause and `collapse` clause, and remove `private` clause?